src/modules/python.rs

```diff
@@ -8,11 +8,15 @@ use super::{Context, Module};
 ///
 /// Will display the Python version if any of the following criteria are met:
 ///     - Current directory contains a `.py` file
-///     - Current directory contains a `.python-version` file
```

Comment:
I'm pretty sure `.python-version` is used by pyenv. 🤔

Reply:
whoops!